pkgs/halo2-binary-merkle-tree/src/lib.rs

```diff
@@ -49,24 +48,6 @@ pub fn generate_merkle_proof(leaves: Vec<String>, leaf: String, depth: usize) ->
         .context("could not serialize merkle proof bytes")?)
 }
 
-// #[cfg(target_arch = "wasm32")]
-// #[wasm_bindgen]
-// pub fn generate_merkle_proof(
```

Comment:
@duguorong009 this version is used in the wasm code for the browser, so you will see that it returns a type of JsValue for the browser if there is an error.

Reply:
Oh, my bad!
I'll roll back & uncomment the code.